metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py

```diff
@@ -1197,6 +1185,17 @@ def get_workspace_workunit(
                 for workunit in dataset_workunits:
                     yield workunit
 
+    def get_workunit_processors(self) -> Sequence[Optional[MetadataWorkUnitProcessor]]:
```

Comment:
Pay attention to this file, some pretty non-standard behavior. Not sure if there's a cleaner way to do this... could directly overwrite `get_workunits` as one option